crates/bevy_scene/src/scene.rs

```diff
@@ -1,4 +1,5 @@
 use bevy_app::AppTypeRegistry;
+use bevy_ecs::reflect::ReflectResource;
 use bevy_ecs::{
     entity::EntityMap,
     reflect::{ReflectComponent, ReflectMapEntities},
```

Comment:
Nit: These could be combined as well.
```suggestion
use bevy_ecs::{
    entity::EntityMap,
    reflect::{ReflectComponent, ReflectMapEntities, ReflectResource},
```